extensions/tika/runtime/src/main/java/io/quarkus/tika/Content.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.tika;
+
+public class Content {
+
+    private String content;
+    private Metadata metadata;
+
+    public Content(String content, Metadata metadata) {
+        this.content = content;
+        this.metadata = metadata;
+    }
+
+    public String getText() {
```

Comment:
The getter should be consistent.

Reply:
@gsmet can you clarify it please ?

Reply:
The getter is called `getText()` whereas the field is called `content`.